google/cloud/spanner_dbapi/connection.py

```diff
@@ -508,7 +508,7 @@ def connect(
     pool=None,
     user_agent=None,
     client=None,
-    route_to_leader_enabled=True,
+    route_to_leader_enabled=False,
```

Comment:
Can we put a comment on top of it mentioning , that ILB should be enabled to enable LAR